packages/asc-ui/src/components/Page/examples/DataportaalFrontpage.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
```

Comment:
Would move this to the `atlas` repository as it's too specific

Reply:
This is the development version, when is ready, we can move it to atlas